client/src/app/shared/utils/array-toString.ts

```diff
@@ -0,0 +1,3 @@
+Array.prototype.toString = function (): string {
+  return this.join(', ');
+};
```

Comment:
let's discuss this approach, according to what happened with prototype.js there are some concerns

Reply:
Agreed

Reply:
Fixed, removed